tiledb/sm/filter/filter_create.cc

```diff
@@ -0,0 +1,71 @@
+//
+// Created by EH on 11/21/2021.
```

Comment:
I guess this should be replaced by the regular @file/section/licence boilerplate

Reply:
Doh!

Done